### ISSUE TITLE
Unable to set temperature in Scheduler Card

### ISSUE THREAD

**Issue by gbuico** [nielsfaber/scheduler-card#531]
### Checklist

- [X] I checked for similar existing issues (both open and closed) before posting.
- [X] I will participate in further discussion about this issue and can help by testing (if requested).

### Card Version

v2.3.3

### Component Version

v3.2.7

### Bug description

I'm using this card/component since many months and it's working fine.... even now.
My schedules are very simple. Same program very day, 3 timeslots with three corresponding temperatures.
I have a common clima unit with three zones. 
The zone can be only set as on/off and the temperature can be set.
When I was editing the card, I was selecting the timeslot and then I had the chance to move a cursor to set the temperature.
Now the scheduler card appears as in bug1.png. "Set temperature" appears two times, both not showing a temperature cursor, "Regulate" is there and has no relation with my units because in allows min/max temperature to be set.
No chance to set a temperature and, apparently, the schedule is empty.
It is not. If I display the scheduler entity, all the valuet I set in the past are there and the scheduler components behave correctly as always. The current schedule entuty can be seen in file bug3.txt attached.
I also attach another file called bug2.txt which is the schedule entity as it was at some point in the past.
I notice that there are some changes, but I cannot swear that bug2.txt is related to a working scheduler card.
I hope this gives you enough information to either identify a bug or let me know what I have to do to fix this
In any case, I'm ready and willing to cooperate to further investigate and provide whatever info you need.
Just for info, the clim is a Daikin ceiling mounted with three flows managed by an Airzone unit.
Thanks for reading till here :)
Giacomo
<img width="325" alt="bug1" src="https://user-images.githubusercontent.com/34986440/153854761-4a3ae161-72c4-42b1-b30b-8e3c641f33f1.png">
[bug2.txt](https://github.com/nielsfaber/scheduler-card/files/8060175/bug2.txt)
[bug3.txt](https://github.com/nielsfaber/scheduler-card/files/8060176/bug3.txt)

### Steps to reproduce

It can be reprocduced just opening the Scheduler card to change one of the settings

**Comment by plouf34** [comment]
same issue for me. When I want to set a temperature, Min and Max variable appears, and next, the scheduling doesn't work.

**Comment by gbuico** [comment]
Update from my side... 
Knowing that it was working before, I redownloaded the oldest available version of this card (2.2.2) and IT WORKED!
I'm attaching here the picture of the working version of the scheduler card (bug4.png
Please note that to make it work I just redownloaded the earlier version without doing any other change. No need to redefine the schedules or to set the temperatures.
At this point I started to upgrade step by step to new versions and determined that the last working one is the 2.3.0 .... 
This means that this problem is caused by the changes introduced by version 2.3.1

Now I went back to 2.3.0 to "survive", but I repeat my offer to cooperate, if needed

Giacomo 




<img width="329" alt="bug4" src="https://user-images.githubusercontent.com/34986440/154049115-f3380b2d-e326-4aa5-ad01-07e95373e6bf.png">

**Comment by nielsfaber** [comment]
It seems that scheduler card is not able to read the settings of your climate entity correctly.
I would never expect the 'regulate' action to show up for a climate which only has on/off mode with temperature setting.
This action applies to A/C units which can do both heating and cooling (i.e. full temperature regulation).
I would expect the 'set temperature' action to show up, with a single slider for the setting.
Could you share a screenshot of the attributes of your climate.livingroom entity (from Developer Tools -> States)?
Thanks.

**Comment by plouf34** [comment]
hvac_modes: heat_cool, off
min_temp: 15
max_temp: 22
current_temperature: 17.6
temperature: 18
current_humidity: 62
hvac_action: idle
icon: mdi:hvac
friendly_name: Ch.Amis
supported_features: 1

**Comment by nielsfaber** [comment]
So your climate entity advertises support for hvac_mode `heat_cool`.
As you can read [here](https://www.home-assistant.io/integrations/climate/), this means it should be set with a low and high temperatures, as the regulate action does.
The set temperature action cannot be used for your entity.

<img src="https://user-images.githubusercontent.com/16861605/156914771-b58b726a-ab04-4925-baa5-e4b367add1cb.png" width="75%" height="75%">

Also, your climate entity is missing attributes `target_temp_low` and `target_temp_high` which are required for entities supporting `heat_cool`.
<img src="https://user-images.githubusercontent.com/16861605/156914846-9c97777e-bb0f-4448-9ab9-b5d3bb04d96d.png" width="75%" height="75%">

In summary, the problems you are experiencing, are caused by the integration of your climate device, not by scheduler.
I cannot help you to fix this.

**Comment by gbuico** [comment]
@nielsfaber 
Her is the screenshot of the climate.livingroom entity...
![image](https://user-images.githubusercontent.com/34986440/156915603-0e140d0b-be42-41ea-aa8d-d73a23ad9763.png)

As you can see, also in my case the equipment advertises for heat_cool but there is no min max temperature to be set.
The heat or cool function is determined by a Daikin internal unit which serves several rooms.
The flow in each room is managed by an Airzone flow controller directly plugged at the exit of the Daikin.
This is the reason why the "main mode" is managed by the Daikin climate entry, while the climate.livingroom can only determine the on/off state for the room and the desired temperature.

Btw, as you have seen above, the schedule card was working ok up to version 2.2.2 ...  afterward it started to propose Regulate and the double Set Temperature (not doing anything if you press on it)
My solution at the moment is to remain at card version 2.2.2 ... but my HA implementation makes a large use of your solution (heating, irrigation etc.) so I'd like to remain on board.
Feel free to ask whatever additional information and thanks for your time
Giacomo

**Comment by nielsfaber** [comment]
@gbuico You have the same situation as plouf34. 
The entity doesn't follow the guidelines defined by HA, see my comment in https://github.com/nielsfaber/scheduler-card/issues/531#issuecomment-1059916446 for more info.
The problem is caused by the Daikin integration, it's best to escalate here.

> As you can see, also in my case the equipment advertises for heat_cool but there is no min max temperature to be set.

This is not allowed by HA, as you can read [here](https://www.home-assistant.io/integrations/climate/#service-climateset_temperature).

It might be the case that a previous version of scheduler card works OK for you, but the changes made in the meantime have improved things for others, so it's hard to unite both worlds.
You could consider to manually add a 'set temperature' action as a workaround, following the procedure [here](https://github.com/nielsfaber/scheduler-card#customizing-built-in-actions) (replace `auto` by `heat_cool`).

**Comment by gbuico** [comment]
@nielsfaber I see your point and I will escalate to the owner of the Airzone integration to see what can be done (the Daikin Unit is managed through the Airzone integration as the control of the Daikin is interfaced dirctly through Airzone).
At the same time I still think that your card has an issue because:
1) during the setup a double "Set Temperature" button appears and cliking on it you deselect the hour range
2) the current integration works perfectly also at the most recent level.... you can't just change the setup

As a side issue, I have difficulties to understand why a climate unit in heat_cool hvac mode may not have a "set temperature".... 
I hope you will do an additional check on this issue.... otherwise I'll find a way
Thank you

**Comment by nielsfaber** [comment]
> during the setup a double "Set Temperature" button appears and cliking on it you deselect the hour range

I believe the 'set temperature' actions are shown since they were configured with an earlier version of the card. I would expect these don't show up if you start with a 'fresh' schedule. The existing schedules may be corrupted due to changes in the mean time.

> As a side issue, I have difficulties to understand why a climate unit in heat_cool hvac mode may not have a "set temperature

This is determined by HA, for service `climate.set_temperature` with only single temperature value it is stated:
> New target temperature for climate device (commonly referred to as a setpoint). 
**Do not use if hvac_mode is heat_cool.**

This card follows the 'rules' set by HA, hence the standard 'set temperature' is hidden for such entities.
